app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1298,6 +1299,19 @@ DeviceDelegateOpenXR::DeleteLayer(const VRLayerPtr& aLayer) {
   }
 }
 
+void DeviceDelegateOpenXR::SwitchPassthrough() {
+  if (!m.passthroughSupported) {
+    VRB_LOG("Passthough not supported");
+    return;
+  }
```

Comment:
Shouldn't this be an assert? I guess it's because you can always click the icon. That made me think that we should not show the icon in the UI if there is no passthrough support, so you'd likely have to implement a new jni method so that Java could query native code about that.

Reply:
Do yo mean we shouldn't show the menu item ? or that it should be shown as dimmed ?

Reply:
I meant that we should not call this method is passthrough is not supported

Reply:
Ok. It seems we need a function to detect from the java part whether the passthough is supported or not. I think @elima will implement it, since he needs it as well for other logic. 

I'd suggest to merge this PR into the the branch @elima is working on, so that we could avoid dealing with rebases, and implement the logic to hide / show the menu item in a follow up patch.

Reply:
The last changes used the DeviceDelegateOpenXR::SupportsPassthrough to hide / show the menu from the java side, via JNI.